app/src/main/res/layout/overview_row.xml

```diff
@@ -0,0 +1,330 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:orientation="vertical" android:layout_width="match_parent"
+    android:minHeight="60dp"
+    android:layout_height="wrap_content"
+    android:background="@color/softGreen"
+    android:paddingTop="@dimen/font_large"
+    android:paddingBottom="@dimen/font_large">
+
+    <TextView
+        android:id="@+id/birthDate"
+        tools:text="X"
+        style="@style/RecyclerViewCell"
+        android:textSize="20sp" />
```

Comment:
Why? `RecyclerViewCell` already has a font size that's consistent for all the other views.